flink-ml-lib/src/main/java/org/apache/flink/ml/common/util/quantile/QuantileSummary.java

```diff
@@ -395,12 +400,14 @@ public QueryResult(int index, long minRankAtIndex, double percentile) {
      *   <li>delta: the maximum span of the rank.
      * </ul>
      */
-    private static class StatsTuple implements Serializable {
+    public static class StatsTuple implements Serializable {
         private static final long serialVersionUID = 1L;
-        private final double value;
+        private double value;
         private long g;
         private long delta;
 
+        public StatsTuple() {}
```

Comment:
Is this change needed to make this class POJO? If yes, then why its member variable is private?

Reply:
No, it isn't. We don't need to make its member variable public because I define the `TypeInfomation` for `StatsTuple` with `Types.POJO`. We need to